packages/components/atoms/f-image-tile/src/components/_tests/ImageTile.test.js

```diff
@@ -7,4 +7,164 @@ describe('ImageTile', () => {
         expect(wrapper.exists()).toBe(true);
     });
+
+    describe('props :: ', () => {
+        describe('isLink :: ', () => {
+            it('should apply `href` to anchor when true', () => {
+                // Arrange
+                const propsData = { isLink: true, href: 'https://www.google.com' };
+
+                // Act
+                const wrapper = shallowMount(ImageTile, {
+                    propsData
+                });
+
+                const link = wrapper.find('[data-test-id="image-tile-link"]');
+
+                // Assert
+                expect(link.attributes('href')).toEqual(propsData.href);
+            });
+
+            it('should not apply `href` to anchor when false', () => {
+                // Arrange
+                const propsData = { isLink: false, href: 'https://www.google.com' };
+
+                // Act
+                const wrapper = shallowMount(ImageTile, {
+                    propsData
+                });
+
+                const link = wrapper.find('[data-test-id="image-tile-link"]');
+
+                // Assert
+                expect(link.attributes('href')).toEqual('#');
+            });
+        });
+
+        describe('tileId :: ', () => {
+            it('should apply `tileId` to the input id', () => {
+                // Arrange
+                const propsData = { tileId: 'chicken' };
+
+                // Act
+                const wrapper = shallowMount(ImageTile, {
+                    propsData
+                });
+
+                const input = wrapper.find('[data-test-id="image-tile-input"]');
+
+                // Assert
+                expect(input.attributes('id')).toContain(propsData.tileId);
+            });
+
+            it('should apply `tileId` to the label for', () => {
+                // Arrange
+                const propsData = { tileId: 'chicken' };
+
+                // Act
+                const wrapper = shallowMount(ImageTile, {
+                    propsData
+                });
+
+                const label = wrapper.find('[data-test-id="image-tile-label"]');
+
+                // Assert
+                expect(label.attributes('for')).toContain(propsData.tileId);
+            });
+
+            describe('imgSrc :: ', () => {
+                it('should apply `tileId` to the label for', () => {
+                    // Arrange
+                    const propsData = { imgSrc: 'https://via.placeholder.com/150' };
+
+                    // Act
+                    const wrapper = shallowMount(ImageTile, {
+                        propsData
+                    });
+
+                    const image = wrapper.find('[data-test-id="image-tile-image"]');
+
+                    // Assert
+                    expect(image.attributes('src')).toContain(propsData.imgSrc);
+                });
+            });
+
+            describe('isSelected :: ', () => {
+                it('should make `isFilterSelected` true when set to true', () => {
```

Comment:
You can use [it.each](https://github.com/justeat/fozzie-components/blob/7c23f7f154ccd83bf9dd7e63e1ca1ed99798dea3/packages/components/pages/f-checkout/src/components/_tests/AgeVerification.test.js#L109-L121) for these type of checks. We have many examples in the repo, like [here](https://github.com/justeat/fozzie-components/blob/7c23f7f154ccd83bf9dd7e63e1ca1ed99798dea3/packages/components/pages/f-checkout/src/components/_tests/AgeVerification.test.js#L109-L121)

Reply:
Awesome, I'll update this 👍